neo4j_utils/_driver.py

```diff
@@ -1566,9 +1565,9 @@ def str_to_exc(e):
 
             return (
                 e.__class__
-                    if isinstance(e, Exception) else
+                if isinstance(e, Exception) else
```

Comment:
in multi-line `if ... else` statements I indent the `if ... else` lines for better readability